kano_updater/apt_wrapper.py

```diff
@@ -355,8 +355,8 @@ def fix_broken(self, progress):
         if self._cache._depcache.broken_count:
             try:
                 self._cache._depcache.fix_broken()
-            except SystemError as err:
-                logger.error(err)
+            except SystemError as e:
+                logger.error('Error attempting to fix broken pkgs {}'.format(e))
```

Comment:
Maybe it would be more useful to do:
```suggestion
                logger.error('Error attempting to fix broken pkgs', exception=e)
```
And similarly below.

Reply:
Good point, Alex. I'll use that instead.